BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -412,9 +412,9 @@ SecStartup (
   // Save NVS data
   NvsData = GetFspNvsDataBuffer (LdrGlobal->FspHobList, &MrcDataLen);
   if ((NvsData != NULL) && (MrcDataLen > 0)) {
-    DEBUG ((DEBUG_INFO, "Save MRC Training Data (0x%p 0x%06X) ... ", NvsData, MrcDataLen));
+    DEBUG ((DEBUG_INFO, "Save MRC Training Data (0x%p 0x%06X) ... \n", NvsData, MrcDataLen));
     SubStatus = SaveNvsData (NvsData, MrcDataLen);
-    DEBUG ((DEBUG_INFO, "%X\n", SubStatus));
+    DEBUG ((DEBUG_INFO, "SaveNvsData SubStatus: %r\n", SubStatus));

```

Comment:
Let us move to a single line DEBUG to save the code size.

Reply:
I meant you have two DEBUG statements.  One before SaveNvsData() and one after.  
If you want to make it clear,  just print the full DEBUG message after the SaveNvsData call.

Reply:
done, combined both prints, and moved after SaveNvsData call.